packages/neuron-ui/src/components/DepositDialog/hooks.ts

```diff
@@ -239,22 +268,37 @@ export const useBalanceReserved = () => {
 
 export const useOnDepositDialogSubmit = ({
   onDepositSuccess,
-  walletID,
+  wallet,
+  multisigConfig,
 }: {
   onDepositSuccess: () => void
-  walletID: string
+  wallet: State.Wallet
+  multisigConfig?: MultisigConfig
 }) => {
   const dispatch = useDispatch()
   return useCallback(() => {
-    dispatch({
-      type: AppActions.RequestPassword,
-      payload: {
-        walletID,
-        actionType: 'send',
-        onSuccess: onDepositSuccess,
-      },
-    })
-  }, [dispatch, walletID, onDepositSuccess])
+    if (multisigConfig) {
+      dispatch({
+        type: AppActions.RequestPassword,
+        payload: {
+          walletID: wallet.id,
+          actionType: multisigConfig.m === 1 ? 'send-from-multisig-need-one' : 'send-from-multisig',
```

Comment:
This can be verified using the `getMultisigSignStatus` result named `canBroadcastAfterSign`.